src/main.c

```diff
@@ -400,6 +414,145 @@ static void fill_grid(char *user_board, char *plain_board, int x_cursor, int y_c
 	}
 }
 
+char* get_paused_file_path(void) {
+	const char* home_path = getenv("HOME");
+	size_t len_home_path = strlen(home_path);
+
+	const char* dir_name = "/.nudoku/";
+	size_t len_dir_name = strlen(dir_name);
+
+	char* dir_path = malloc(len_home_path + len_dir_name + 1);
+	strcpy(dir_path, home_path);
+	strcat(dir_path, dir_name);
+
+	struct stat st = {0};
+    if (stat(dir_path, &st) == -1) {
+        mkdir(dir_path, 0700);
+    }
+
+	const char* file_name = "game_state.txt";
+	size_t len_file = strlen(file_name);
+
+	char* file_path = malloc(len_home_path + len_dir_name + len_file + 1);
+	strcpy(file_path, dir_path);
+	strcat(file_path, file_name);
+	
+	free(dir_path);
+	return file_path;
```

Comment:
When is this file removed again? I think in `is_game_saved` you test for the presence of it. But I don't see where it get's removed when the game is finished.